pkg/api/v20220904/secret_convert.go

```diff
@@ -0,0 +1,39 @@
+package v20220904
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"github.com/Azure/ARO-RP/pkg/api"
+)
+
+type secretConverter struct{}
+
+func (c secretConverter) ToExternal(s *api.Secret) interface{} {
```

Comment:
It looks like Microsoft doesn't want secrets to be retrievable. I'm getting an error from their linter: Secret property "secretResources" cannot be sent in the response

https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/Semantic-and-Model-Violations-Reference.md#SECRET_PROPERTY

Reply:
I've removed `SecretResources` from the `ToExternal` secret converter.